topics/variant-analysis/tutorials/MTB_transmission/tutorial.md

```diff
@@ -0,0 +1,810 @@
+| ERR5987300   | -          | Pre-XDR    | Unclustered |
+| ERR6362078   | -          | MDR        | Unclustered |
+| ERR6362139   | -          | Pre-MDR    | Unclustered |
+| ERR6362333   | -          | Pre-XDR    | Unclustered |
+| ERR6362653   | -          | MDR        | Unclustered |
+| SRR13046689  | -          | Other      | Unclustered |
+| SRR998584    | -          | Sensitive  | Unclustered |
+| ERR5987352   | 10         | Pre-MDR    | Clustered   |
+| ERR6362484   | 10         | Pre-MDR    | Clustered   |
+| ERR6362138   | 12         | MDR        | Clustered   |
+| ERR6362156   | 12         | Pre-XDR    | Clustered   |
+| ERR6362253   | 12         | MDR        | Clustered   |
+
+
+
+> ### {% icon question %} Question
+>
+> Assuming that we have a very good sampling of the outbreak. Which strains **may** represent instances
+> of *de novo* evolution of drug resistance and which ones instances of *transmitted* (primary) resistance?
+> Remember that you can look at the TB-profiler reports of independent samples for detailed information.
+>
+> > ### {% icon solution %} Solution
+> > In a simplistic scenario, we could consider clustered strains as instances of transmission and
+> > unclustered strains as instances of de novo evolution of DR. Thus, we see that for example there
+> > are three MDR strains (ERR551620, ERR6362078, ERR6362653) that are unclustered and therefore may
+> > represent cases in which drug resistance evolved independently as response to treatment within the respective
+> > patients. However you need to always bear in mind that, although **within our population** those MDR
+> > strains doesn't seem to be linked to transmission, this does NOT rule out the possibility that
+> > some of these patients were infected with an MDR strain somewhere else.
+> >
+> > When looking at clustered strains, distinguishing between transmitted and *de-novo* may be tricky. Note
+> > that, for example, for the two RIF-monoresistant strains linked within the same transmission
+> > cluster there are, at least, a couple of scenarios possibe: one in which a RIF-monoresistant strain
+> > evolved in one patient and was transmitted to the other patient afterwards, and one in which both
+> > were infected with the same RIF-monoresistant strain from a third patient that we have not sampled.
+> > We need to note that, in the first scenario, drug resistance evolved *de novo* in one patient,
+> > and was *transmitted* to the other patient, whereas in the second scenario drug resistance was
+> > *transmitted* in both cases.
+> {: .solution}
+{: .question}
+
+
+
+> ### {% icon question %} Question
+>
+> 1. The same principles than those explained above apply to the three MDR strains that are
+> within the same transmission cluster. However in this case there is one strain that shows clear
+> evidence of *de-novo* evolution of DR. Do you know which strain and why?
+> 2. Are there possible scenarios other than *de-novo* evolution of DR for this strain?
```

Comment:
is this question unanswered? I only see a 1 below

Reply:
Yes you are right. I have removed this question as it did not add that much.